app/main.py

```diff
@@ -295,7 +296,10 @@ def extract_thumbnail_from_xml_file():
     query_args = request.args
     if 'path' not in query_args:
         return abort(400, description=f"Query arguments are not valid.")
+    # if 's3BucketName' not in query_args:
+    #     return abort(400, description=f"Query arguments must include 's3BucketName'.")
 
+    s3BucketName = query_args.get("s3BucketName") or Config.S3_BUCKET_NAME
```

Comment:
I think it is better to get from a dictionary like this in Python: `query_args.get("s3BucketName", Config.S3_BUCKET_NAME)`